sqlmesh/dbt/manifest.py

```diff
@@ -256,6 +256,7 @@ def _load_manifest(self) -> Manifest:
             profiles_dir=str(self.profiles_path),
             target=self.target.name,
             macro_debugging=False,
+            REQUIRE_RESOURCE_NAMES_WITHOUT_SPACES=True,
```

Comment:
I assume this is backwards compatible?

Reply:
That's the expectation, yes. If the build passes that would be the confirmation of that.